tools/myst/host/utils.c

```diff
@@ -349,3 +350,148 @@ void free_mount_mapping_opts(myst_mount_mapping_t* mappings)
         free(mappings->mounts);
     }
 }
+
+int create_roothash_file(const char* path, char filename[PATH_MAX])
```

Comment:
Suggest renaming to `read_roothash_from_ext2_rootfs`. Again, can we skip saving the roothash to a file?

Reply:
I eliminated saving of roothashes to file in favor of a new "roothashes" binary region, so that we no longer need the temporary file to do CPIO packaging when creating the "archive" region.